packages/docusaurus-theme-search-algolia/src/theme/SearchPage/index.js

```diff
@@ -169,25 +239,19 @@ function Search() {
       : 'Search the documentation';
 
   const makeSearch = (page = 0) => {
-    if (searchVersion) {
-      algoliaHelper
-        .setQuery(searchQuery)
-        .addFacetRefinement('version', searchVersion)
-        .setPage(page)
-        .search();
-    } else {
-      algoliaHelper.setQuery(searchQuery).setPage(page).search();
-    }
-  };
-
-  const handleSearchInputChange = (e) => {
-    const searchInputValue = e.target.value;
+    algoliaHelper.setQuery(searchQuery).setPage(page);
+
+    algoliaHelper.addDisjunctiveFacetRefinement('docusaurus_tag', 'default');
+    Object.entries(docsSearchVersionsHelpers.searchVersions).forEach(
+      ([pluginId, searchVersion]) => {
+        algoliaHelper.addDisjunctiveFacetRefinement(
+          'docusaurus_tag',
+          `docs-${pluginId}-${searchVersion}`,
+        );
+      },
+    );
```

Comment:
query is like: `default OR docs-alpha66 OR docs-community-current`, ie we search in pages, blogs, and the selected docs versions .

It's an improvement compared to previous search page as we  can now search in pages and blog, which was not possible by using the previous facet filter logic (only searching in docs)

see:
- before: https://v2.docusaurus.io/search?q=yangshun
- after: https://deploy-preview-3604--docusaurus-2.netlify.app/classic/search?q=yangshun